pkg/annotation/k8s.go

```diff
@@ -117,6 +117,14 @@ const (
 	ServiceAffinity      = ServicePrefix + "/affinity"
 	ServiceAffinityAlias = Prefix + "/service-affinity"
 
+	// ServiceLoadBalancingAlgo indicates which backend selection algorithm for
+	// a given Service to use. This annotation will override the default value
+	// set in bpf-lb-algorithm.
+	// Allowed values:
+	// - random
+	// - maglev
+	ServiceLoadBalancingAlgo = ServicePrefix + "lb-algorithm"
```

Comment:
Did you try this? Shouldn't this be: `+ "/lb-algorithm"`

Reply:
fixed.
I tested this change with variable `ServiceLoadBalancingAlgo` so it passed :)